Source/Core/Core/PowerPC/Jit64/Jit_Integer.cpp

```diff
@@ -50,14 +50,19 @@ void Jit64::GenerateOverflow()
 	SetJumpTarget(exit);
 }
 
+bool Jit64::MergeAllowedNextInstruction()
+{
+	// Be careful: a breakpoint kills flags in between instructions
+	return PowerPC::GetState() != PowerPC::CPU_STEPPING && !js.isLastInstruction && !js.next_inst_bp && !js.next_op->isBranchTarget;
+}
```

Comment:
Note: the MergeAllowedNextInstruction bits are important to merge even if the rest of the PR is rejected for some reason.